package-manifest.toml

```diff
@@ -638,10 +638,10 @@ source.repo = "maghemite"
 # `tools/maghemite_openapi_version`. Failing to do so will cause a failure when
 # building `ddm-admin-client` (which will instruct you to update
 # `tools/maghemite_openapi_version`).
-source.commit = "93cd0d642cf1b58f9f4528275e2a2aa758e9feb3"
+source.commit = "8da5bd1c4e41c9b5cc2513eec443e7d761c9f5e5"
```

Comment:
I think this commit should have been in sync with the other two later in the file, so I updated all three to point to the latest `main`. (@sunshowers please correct me if I'm wrong - it looks like they diverged on #7516?)

Reply:
So uh, to be honest, I tried getting to all of them but completely missed this...

I feel like we should have one place to define versions and hashes if at all possible. Maybe in a different section in this file, or in another file.